apps/aecore/src/aec_blocks.erl

```diff
@@ -420,7 +420,6 @@ validate_key_block(#key_block{} = Block) ->
 validate_micro_block(#mic_block{} = Block) ->
     Validators = [fun validate_txs_hash/1,
                   fun validate_gas_limit/1,
-                  fun validate_txs_fee/1,
```

Comment:
I see that the `aec_conductor` inside the DB transaction attempting to insert microblock in DB [checks this by relying on `aetx:check`](https://github.com/aeternity/epoch/blob/e0bfd9e4901a0c1a98c89c172f54f9d726475e35/apps/aecore/src/aec_chain_state.erl#L816). So block invalid for low fee is not inserted in DB anyway, that is good.

I see that this check function used to be run in the process performing the sync network operation (see `aec_conductor:add_synced_generation_batched`) while now the first time this check is run on the microblock is inside the `aec_conductor` process that means this may lead to network peer potential giving more load to `aec_conductor` process. (Similar code path applies for user HTTP API `aec_conductor:post_block` though that is meant to be internal so less of a concern.)